pipelines/build/common/build_base_file.groovy

```diff
@@ -664,19 +664,40 @@ class Builder implements Serializable {
         return "jdk${number}"
     }
 
+
+    /*
+    Returns the downstream build job's type by checking job folder's path
+    either it is "prototype" or "release" or the rest (can be nightly or pr-tester)
```

Comment:
I'm not sure about this wording, are we saying the enum is "prototype", "release", "nightly" and "pr-tester"?

Reply:
did updates for the comments